wrphttp/headers.go

```diff
@@ -57,14 +80,17 @@ var (
 func getMessageType(h http.Header) wrp.MessageType {
 	value := h.Get(MessageTypeHeader)
 	if len(value) == 0 {
-		panic(errMissingMessageTypeHeader)
+		// check alternative header prefix
+		value = h.Get(msgTypeHeader)
+		if len(value) == 0 {
```

Comment:
```suggestion
		if value == "" {
```

Reply:
As Wes pointed out, the rest of the around this also uses `len(string) == 0` and consistency is better at the moment. We will leave this conditional as it is.